security/pkg/nodeagent/sds/sdsservice_test.go

```diff
@@ -70,6 +71,46 @@ func TestStreamSecretsForWorkloadSds(t *testing.T) {
 	testHelper(t, arg, sdsRequestStream, false)
 }
 
+//The purpose of adding these tests is to verify that SDS agent
```

Comment:
Revise the comment? 
e.g. // Verifies that SDS agent is using a valid token returned by credential fetcher and pushing SDS resources back successfully.